core/deepmind.py

```
@@ -503,6 +503,12 @@ def _is_missing_user_identity(sender_id: str, sender_name: str) -> bool:
         invalid_names = {"", "用户", "成员", "未知", "未知用户", "user", "unknown", "none", "null"}
         sid = str(sender_id or "").strip()
         name = str(sender_name or "").strip()
+        
+        # 核心逻辑改进：只要有合法的 ID，就认为身份有效。
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)